docs/user/ppl/cmd/streamstats.rst

```diff
@@ -0,0 +1,476 @@
+=============
+streamstats
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| The ``streamstats`` command is used to calculate cumulative or rolling statistics as events are processed in order. Unlike ``stats`` or ``eventstats`` which operate on the entire dataset at once, it computes values incrementally on a per-event basis, often respecting the order of events in the search results. It allows you to generate running totals, moving averages, and other statistics that evolve with the stream of events.
+
+| Key aspects of `streamstats`:
```

Comment:
`|` at the beginning of line means starting a new line. So no need to add empty new line if you add `|` at the beginning of line